packages/perseus/src/styles/widgets/image.less

```diff
@@ -52,3 +52,24 @@
         margin-top: 5px;
     }
 }
+
+// Image widgets are centred, by default
+.perseus-image-widget {
+    margin-inline-start: auto;
+    margin-inline-end: auto;
```

Comment:
I think this is more confusing than it needs to be. I'm guessing it's for right-to-left? But we don't support RTL in any other capacity. I'm also not sure [it works](https://caniuse.com/?search=margin-inline-start) in all the [browsers we support](https://khanacademy.atlassian.net/wiki/spaces/ENG/pages/103612417/Browser+Support).